@types/pal/minigame.d.ts

```diff
@@ -23,7 +23,7 @@ declare module 'pal/minigame' {
          * @returns {SafeArea} An interface displaying the data of safe area, including 'top', 'bottom', 'left', 'right', 'width' and 'height'.
          */
         getSafeArea(): SafeArea;
-        triggerGC(): void;
+        triggerGC?(): void;
```

Comment:
What does it mean?

Reply:
It means that the API exists or does not exist on this platform.